superset-frontend/cypress-base/cypress/integration/dashboard/edit_mode.test.js

```diff
@@ -23,47 +23,54 @@ describe('Dashboard edit mode', () => {
     cy.server();
     cy.login();
     cy.visit(WORLD_HEALTH_DASHBOARD);
-    cy.get('.dashboard-header [data-test=edit-alt]').click();
+    cy.get('[data-test="dashboard-header"]')
+      .find('[data-test=edit-alt]')
+      .click();
   });
 
-  xit('remove, and add chart flow', () => {
+  it('remove, and add chart flow', () => {
     // wait for box plot to appear
-    cy.get('.grid-container .box_plot');
-
-    cy.get('.fa.fa-trash')
+    cy.get('[data-test="grid-container"]').find('.box_plot', {
+      timeout: 10000,
+    });
+    cy.get('[data-test="dashboard-component-chart-holder"]')
+      .find('[data-test="dashboard-delete-component-button"]')
```

Comment:
Sorry if I've asked this before, but I'm curious why we need to `get` AND `find` here. If `dashboard-delete-component-button` is specific enough, it seems like we could just `get` that directly.

Reply:
To reflect structure of component - I just want to show in test that this delete button belongs to chart holder - it is nothing to make tests more stable - just more strict.